apps/explorer/src/components/Object/utils.ts

```diff
@@ -51,37 +51,60 @@ export function extractSerializationType(type: SuiMoveNormalizedType | ''): Type
 	return type;
 }
 
-export function getFieldTypeValue(type: SuiMoveNormalizedType | '', objectType: string) {
+function getDisplayName(type: SuiMoveNormalizedType | '', objectType: string) {
 	const normalizedType = extractSerializationType(type);
+
 	if (typeof normalizedType === 'string') {
-		return {
-			displayName: normalizedType,
-			normalizedType: normalizedType,
-		};
+		let parentKey = '';
+		if (typeof type === 'object') {
+			if ('Vector' in type) {
+				parentKey = 'Vector';
+			} else if ('Reference' in type) {
+				parentKey = 'Reference';
+			} else if ('MutableReference' in type) {
+				parentKey = 'MutableReference';
+			} else if ('TypeParameter' in type) {
+				parentKey = 'TypeParameter';
```

Comment:
What about the else case here?

Reply:
else `parent === ''` and it will just return `normalizedType` on line 70

Reply:
Do we want to explicitly do `parent = ''` again on the else?